tests/display.rs

```diff
@@ -187,15 +319,15 @@ mod structs {
 
             #[derive(Display)]
             #[display(
-            "{_0} {ident} {_1} {} {}",
-            _1, _0 + _1, ident = 123, _1 = _0,
+                "{_0} {ident} {_1} {} {}",
+                _1, _0 + _1, ident = 123, _1 = _0,
```

Comment:
I think we should have tests for delegation with multi field structs too. i.e. this should trigger our delegation logic too (but on the second field)
```rust
#[derive(Display)]
#[display("{_1}")]
struct TupleDisplay(i32, i32);
```